Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
+    commander (4.5.2)
       highline (~> 2.0.0)
-    database_cleaner (1.7.0)
-    diff-lcs (1.3)
+    database_cleaner (1.8.5)
```

Comment:
### [database_cleaner](https://github.com/DatabaseCleaner/database_cleaner)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.7.0 → 1.8.5

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **218** commits. See the full changes on [the compare page](https://github.com/DatabaseCleaner/database_cleaner/compare/v1.7.0...v1.8.5).

These are the first 10 commits:
- (c0013de) [add a url whitelist](https://github.com/DatabaseCleaner/database_cleaner/commit/c0013de1ee7afb4e25a0581940e22408f73273f9)
- (b89590b) [Adding tests](https://github.com/DatabaseCleaner/database_cleaner/commit/b89590bf1863579de3496578980346728be970d0)
- (8e34c1e) [fix travis.](https://github.com/DatabaseCleaner/database_cleaner/commit/8e34c1e6521f3e76ce61b6be49d74ce03720fc1a)
- (7519d43) [consider all sqlite DATABASE_URLs local.](https://github.com/DatabaseCleaner/database_cleaner/commit/7519d4381073dbcdda732a09d7c92a70fe312f8c)
- (fc3ac68) [workaround old mysql/mysql2 adapter incompatibility with MyS…](https://github.com/DatabaseCleaner/database_cleaner/commit/fc3ac68474c1e7bb5c24ca0829edd223d53162ea)
- (ad901d6) [fix race condition in ensure_counts assertion.](https://github.com/DatabaseCleaner/database_cleaner/commit/ad901d631ec64a89db2a109eaf0d779b31198b9a)
- (df330de) [fix race condition where moped can return before deletions a…](https://github.com/DatabaseCleaner/database_cleaner/commit/df330de4c829d4a43ec95cafd98b649c2710501d)
- (fafca34) [Merge pull request #532 from botandrose/mysql-5.7.3](https://github.com/DatabaseCleaner/database_cleaner/commit/fafca34e70103f6ee55bcae722df6e7cb0ceedc9)
- (eeb56d7) [Merge pull request #535 from botandrose/fix_moped_race_condi…](https://github.com/DatabaseCleaner/database_cleaner/commit/eeb56d7bf6feeb6f55371386502fbe4bd13f86a1)
- (f3feaa3) [Use count(*) > 0 instead of count(*)](https://github.com/DatabaseCleaner/database_cleaner/commit/f3feaa36ba463a558ae7c115a7ebe60ea08a5e7f)


</details>